api/v1alpha1/variantautoscaling_types.go

```diff
@@ -18,6 +18,12 @@ type VariantAutoscalingSpec struct {
 	// ModelProfile provides resource and performance characteristics for the model variant.
 	// +kubebuilder:validation:Optional
 	ModelProfile ModelProfile `json:"modelProfile"`
+
+	// VariantCost specifies the cost per replica for this variant (used in capacity analysis).
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:validation:Pattern=`^\d+(\.\d+)?$`
+	// +kubebuilder:default="10.0"
+	VariantCost string `json:"variantCost,omitempty"`
```

Comment:
We already have a cost field in the accelerator configMap. Do we need another field here in the VA spec?